CMakeLists.txt

```diff
@@ -19,15 +19,15 @@ function(compiler_not_supported message)
 endfunction(compiler_not_supported)
 
 if("${CMAKE_CXX_COMPILER_ID}" STREQUAL "GNU")
-    if (CMAKE_CXX_COMPILER_VERSION VERSION_LESS 8.2)
+    if (CMAKE_CXX_COMPILER_VERSION VERSION_LESS 9.0)
```

Comment:
If I am not mistaken Ubuntu 19.04 provides packages for GCC 9.1 and 19.10 directly ships with 9.2.